src/tsickle.ts

```diff
@@ -1222,6 +1222,11 @@ class Annotator extends ClosureRewriter {
     const forwardDeclarePrefix = `tsickle_forward_declare_${++this.forwardDeclareCounter}`;
     const moduleNamespace =
         nsImport !== null ? nsImport : this.host.pathToModuleName(this.file.fileName, importPath);
+    const moduleSymbol = this.typeChecker.getSymbolAtLocation(specifier);
+    // Modules that do not have exports do not have a symbol. These can still be imported, either as
```

Comment:
This is a surprise to me!  But I guess it sorta makes sense.

In your test it looks like it's importing a script, not a module.  Is it worth adding an "export {}" to make it a module?  I think the answer is no, because people are likely to forget that (or intentionally leave it out) on files imported for their side effects.

So I think my suggestion here is to just adjust this comment to also mention scripts.